server/src/test/java/org/apache/druid/metadata/IndexerSQLMetadataStorageCoordinatorTest.java

```diff
@@ -3050,19 +3106,20 @@ private List<DataSegment> createAndGetUsedYearSegments(final int startYear, fina
 
   private ImmutableList<DataSegment> retrieveUnusedSegmentsUsingMultipleIntervalsAndLimit(
```

Comment:
Should this method be renamed to include offset in its name, or perhaps could use a simpler name?

Reply:
fixed